LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (C) 2017, 2018, 2019, 2020, 2021, 2022 CERN.
+Copyright (C) 2017, 2018, 2019, 2020, 2021, 2022, 2023 CERN.
```

Comment:
Please edit also `docs/conf.py` where we have `2017-2020` and use `2017-2023` there.

Reply:
BTW it is not necessary to split this in two commits; the addition of a new author and the update of the copyright year can come with the same commit as the feature.  (So that people wouldn't wonder who this new comitter is.)